src/prx/helpers.py

```diff
@@ -126,18 +124,34 @@ def rinex_header_time_string_2_timestamp_ns(time_string: str) -> pd.Timestamp:
 
 
 def repair_with_gfzrnx(file):
+    path_folder_gfzrnx = prx_repository_root().joinpath("tools", "gfzrnx")
     gfzrnx_binaries = glob.glob(
-        str(prx.helpers.prx_repository_root() / "tools/gfzrnx/**gfzrnx**"),
-        recursive=True,
+        "**gfzrnx**",
+        root_dir=path_folder_gfzrnx
     )
     assert len(gfzrnx_binaries) > 0, "Could not find any gfzrnx binary"
     for gfzrnx_binary in gfzrnx_binaries:
-        command = f" {gfzrnx_binary} -finp {file} -fout {file}  -chk -kv -f"
-        result = subprocess.run(command, capture_output=True, shell=True)
-        if result.returncode == 0:
-            log.info(f"Ran gfzrnx file repair on {file}")
-            return file
-    assert False, f"gdzrnx file repair run failed: {result}"
+        command = [str(path_folder_gfzrnx.joinpath(gfzrnx_binary)),
+                   "-finp", str(file),
+                   "-fout", str(file),
+                   "-chk", "-kv",
+                   "-f",
+                   ]
+        try:
+            result = subprocess.run(
+                command,
+                capture_output=True,
+                shell=True,
+            )
+            if result.returncode == 0:
+                log.info(f"Ran gfzrnx file repair on {file}")
+            else:
+                log.info(f"gfzrnx file repair run failed: {result}")
+        except OSError:
```

Comment:
Shouldn't we fail early when this happens, as other stuff can break downstream as a consequence of this step failing?

Reply:
Ok, I returned to strict execution check. Any failed execution of `gfzrnx` will stop the program.